PennMobile/src/main/java/com/pennapps/labs/pennmobile/GsrRoomHolder.kt

```diff
@@ -0,0 +1,27 @@
+package com.pennapps.labs.pennmobile
+
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.TextView
+import kotlinx.android.synthetic.main.gsr_room.view.*
+
+class GsrRoomHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    internal var gsrRoom: LinearLayout? = null
```

Comment:
Not completely sure but I think this could be made a nonnullable lateinit var. Honestly, these variables might not be necessary because you can just call `itemView.gsr_room` etc.